db/migrate/20151107212817_add_paid_at_to_payment.rb

```diff
@@ -0,0 +1,6 @@
+class AddPaidAtToPayment < ActiveRecord::Migration
+  def change
+    add_column :payments, :paid_at, :datetime
+    remove_column :payments, :state
```

Comment:
For a rollback to work here I think you would need to do both `up` and `down` so you can set the column type of `payments`.